the-association/About-us.rst

```diff
@@ -0,0 +1,71 @@
+About the OCA
+=============
+
+The Odoo Community Association, or OCA, is a nonprofit organization whose
+mission is to support the collaborative development of the Odoo features and to
+promote the widespread use of Odoo. The Association provides financial,
+organizational and legal support to the broader Odoo Open Source community.
+It also serves as an independent legal entity to which community members can
+contribute to the code, the funding and other resources, with the knowledge
+that their contributions will be maintained for public benefit.
+
+**The Association’s projects are all freely available and usable under an
+OSI-certified Open Source license.**
+
+Our Goals
+=========
+
+* Help and **promote** the collaborative software development of Odoo;
+* Encourage the development of Odoo and its features while **coordinating** and
+  organizing the collaborative work on the software;
+* Assist the community while **defending its interests** and the sustainability
+  of its developments;
+* **Promote** the use of the Odoo solution;
+* Facilitate synergies, collaborations and **fund raising** efforts;
+* Actively **collaborate** on the definition of the roadmaps of new versions of
+  the tool and their implementation.
+
+Mission Statement
+=================
+
+Establish and support an Open Source and collaborative community for the
+development and promotion of the Odoo features and modules.
+
+Objectives
+==========
+
+Specific goals support the overall mission:
+
+* To ensure a high degree of quality in the OCA’s projects in order to build
+  and preserve the association “brand”.
+* To ensure the long term support and maintenance of the OCA’s projects and
+  developments.
+* To provide resources for the OCA’s projects - eg. infrastructure, CLA,
+  funding, legal.
+* To encourage the implementation of open standards and standards-based
+  interoperability in the OCA’s projects.
+* To support the worldwide use and contribution to the OCA’s projects from the
+  community via the internationalization of the software and community outreach.
+* To represent the Community’s opinion regarding the Odoo SA roadmap, marketing
+  and development strategies.
+* To promote the use of Open Source software in the industry (not just
+  association software) - eg. training, outreach.
+* To organize members meetings to improve the collaboration and the knowledge of
+  the OCA’s projects.
+
+Founding Members
+================
+
+The Odoo Community Association has been launched in 2013 by the following
+founding members:
+
+* `Akretion <https://www.akretion.com>`_
+* `Camptocamp <https://www.camptocamp.com>`_
+* `Therp <https://www.therp.nl>`_
+* `Savoir-faire Linux <https://www.savoirfairelinux.com>`_
+* `Vauxoo <https://www.vauxoo.com>`_
+
+All of the founding members are strongly involved in the community and shared
+the same vision about `Collaborative software development
+<http://en.wikipedia.org/wiki/Collaborative_software_development_model>`_.
```

Comment:
Shouldn't we put here all the board history?

Reply:
Actually we should publish here all the AGA reports, year by year